api/server.py

```diff
@@ -1102,20 +1108,36 @@ def start_api_server():
     """
     # Starting the API
     my_api_configuration = api_configuration()
+    api_access_key = my_api_configuration["api_access_key"]
+    api_access_without_key = my_api_configuration["api_access_without_key"]
+
     write_to_api_console(
         " * API access key: {0}\n".format(
-            my_api_configuration["api_access_key"] if not my_api_configuration["api_access_without_key"]
-            else "NOT REQUIRED!"
+                api_access_key
+                if not api_access_without_key
+                else "NOT REQUIRED!"
         )
     )
-    global app
+
     app.config["OWASP_HONEYPOT_CONFIG"] = {
-        "api_access_key": my_api_configuration["api_access_key"],
-        "api_client_white_list": my_api_configuration["api_client_white_list"]["enabled"],
-        "api_client_white_list_ips": my_api_configuration["api_client_white_list"]["ips"],
-        "api_access_log": my_api_configuration["api_access_log"]["enabled"],
-        "api_access_log_filename": my_api_configuration["api_access_log"]["filename"],
-        "api_access_without_key": my_api_configuration["api_access_without_key"],
+        "api_access_key":
+        api_access_key,
+
+        "api_client_white_list":
+        my_api_configuration["api_client_white_list"]["enabled"],
+
+        "api_client_white_list_ips":
+        my_api_configuration["api_client_white_list"]["ips"],
+
+        "api_access_log":
+        my_api_configuration["api_access_log"]["enabled"],
+
+        "api_access_log_filename":
+        my_api_configuration["api_access_log"]["filename"],
+
+        "api_access_without_key":
+        api_access_without_key,
+
```

Comment:
i think best to view a json is 
```json
{
   "name": "value"
}
```

is there any reason you changed it? did it exceed the max length?

Reply:
Yes. I was exceeding the maximum length.